main/lregex.c

```diff
@@ -2753,10 +2753,10 @@ static void   guestRequestSubmit (struct guestRequest *r)
 {
 	const char *langName = getLanguageName (r->lang);
 	verbose ("guestRequestSubmit: %s; "
-			 "range: %ld - %ld\n",
+			 "range: %lld - %lld\n",
```

Comment:
MinGW complains about "%lld". Strange.

https://ci.appveyor.com/project/universalctags/ctags/builds/27974782/job/qclxucel15nj4ufp#L676
```
main/lregex.c: In function 'guestRequestSubmit':
main/lregex.c:2755:11: warning: unknown conversion type character 'l' in format [-Wformat=]
 2755 |  verbose ("guestRequestSubmit: %s; "
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~
main/lregex.c:2756:15: note: format string is defined here
 2756 |     "range: %lld - %lld\n",
      |               ^
```

Reply:
Hmm, it seems that MinGW's `-Wformat` doesn't support "%lld" for compatibility for very old versions of msvcrt.dll:
https://sourceforge.net/p/mingw/bugs/2190/
It might be better to use `PRId64` instead.